packages/dlog/src/tests/dlog.test.ts

```diff
@@ -59,7 +59,7 @@ describe('dlogTest', () => {
     })
 
     test('extend', () => {
-        const base_log = dlog('test:dlog')
+        const base_log = dlog('test:dlog', { useWinston: false })
```

Comment:
do these tests pass with winston? if so you can do a matrix test
```
test.each([true, false]), (useWinston: boolean) => { ... })
```

Reply:
well, ...kinda?! 
we're using our dlog formatting function under the hood with winston, so it handles all the stuff: map, set, bignumbers, etc

but I wasn't able to get the output of winston as a string. And wasn't able to capture the stdout and run tests against it too. Got skill issue'd 😅